packages/shared/src/lib/featureManagement.ts

```diff
@@ -103,7 +103,7 @@ export class Features<T extends FeatureValue = string> {
 
   static readonly CompanionPermissionDescription = new Features(
     'companion_permission_description',
-    "We'll ask for extra permissions so we can show the companion directly on an article!",
+    "We'll ask for extra permissions so we can show the companion directly on an post!",
```

Comment:
We'll ask for extra permissions so we can show the companion directly on the original content!